rebase.sh

```diff
@@ -212,8 +212,6 @@ resolve_conflicts() {
       apply_code_package_changes
     elif [[ "$conflictingFile" == "code/product.json" ]]; then
       apply_code_product_changes
-    elif [[ "$conflictingFile" == "code/extensions/package.json" ]]; then
-      apply_package_changes_by_path "code/extensions/package.json"
```

Comment:
https://github.com/che-incubator/che-code/pull/152#pullrequestreview-1188068551

Reply:
agree, now we can remove it